spectre_analyses/tasks.py

```diff
@@ -0,0 +1,140 @@
+from contextlib import contextmanager
+from functools import partial
+import json
+import os
+import pickle as pkl
+import shutil
+import sys
+
+import celery
+from celery.utils.log import get_task_logger
+
+import spdata.reader
+import spdata.types as ty
+
+from spectre_analyses.celery import app
+import matlab_hooks as mh
+
+
+FILESYSTEM_ROOT = os.path.abspath(os.sep)
+DATA_ROOT = os.path.join(FILESYSTEM_ROOT, 'data')
+STATUS_PATHS = {
+    'all': FILESYSTEM_ROOT,
+    'done': os.path.join(FILESYSTEM_ROOT, 'data'),
```

Comment:
**Minor**
May be shrunk to `DATA_ROOT`.

Reply:
Good point!